src/Uno.UWP/ApplicationModel/Contacts/Contact.cs

```diff
@@ -60,33 +63,108 @@ public string DisplayName
 		/// <summary>
 		/// Gets info about the phones for a contact.
 		/// </summary>
-		public IList<ContactPhone> Phones { get; internal set; } = new NonNullList<ContactPhone>();
-
+		public IList<ContactPhone> Phones { get; internal set; } = new NonNullList<ContactPhone>();
+
+		private string _firstName = "";
 		/// <summary>
 		/// Gets and sets the first name for a contact.
 		/// </summary>
-		public string FirstName { get; set; } = "";
-
+		public string FirstName
+		{
+			get => _firstName;
+			set
+			{
+				_firstName = value;
+				if (_firstName.Length > 64)
```

Comment:
This may fail is _firstName is null, same for the other properties. Add `#nullable enable` at the top of the file.

This also repetitive throughout the file, make a method that validates this instead e.g. `ValidateLength("Windows.ApplicationModel.Contacts.Contact.FirstName", 64)`

Reply:
a) how _firstName can be null? It is initialized as "".
b) I don't know how implement such ValidateLength method (with property name).